packages/cbor/package.json

```diff
@@ -1,10 +1,21 @@
 {
   "description": "Implementation of Concise Binary Object Representation (CBOR) data format (RFC7049).",
   "devDependencies": {
+    "@types/jasmine": "3.4.6",
+    "@types/karma": "3.0.4",
     "@types/node": "~12",
     "cross-env": "6.0.3",
     "istanbul": "1.1.0-alpha.1",
     "jasmine": "3.5.0",
+    "karma": "4.4.1",
+    "karma-chrome-launcher": "3.1.0",
+    "karma-coverage": "2.0.1",
+    "karma-jasmine": "2.0.1",
+    "karma-jasmine-diff-reporter": "2.0.0",
```

Comment:
Many of these plugins are not needed. Check the following link for reference: http://benny.work/2019/04/29/8/